README.md

```diff
@@ -4,7 +4,11 @@ You want to use SHACL shapes to validate RDF-based data but find it hard to go b
 
 SHACL playground allows you to define Data Shapes according to the W3C [Shapes Constraint Language (SHACL)](https://www.w3.org/TR/shacl/) and test them against sample data.
 
-The implementation of the validation is 100% in JavaScript and thus 100% client-side. We provide an instance of our validator at [https://shacl-playground.zazuko.com/](https://shacl-playground.zazuko.com/)
+The implementation of the validation is 100% in JavaScript and thus 100% client-side. We provide an instance of our validator at [https://shacl-playground.zazuko.com/](https://shacl-playground.zazuko.com/).
+
+To run it locally, either start a webpack dev server with `yarn run start` and access it at <http://localhost:8080>, or create a production build with `yarn run build` and serve the created `dist` directory, e.g. with nginx.
```

Comment:
Thanks for adding instructions for running it!

To be complete, can you just add that we should run the `yarn` command to install the dependencies?

Can you remove the `<>` around the URL here? It would be auto-linked without them.